pre_commit/commands/hook_impl.py

```diff
@@ -188,6 +190,7 @@ def _run_ns(
         return _ns(hook_type, color, commit_msg_filename=args[0])
     elif hook_type in {'pre-merge-commit', 'pre-commit'}:
         return _ns(hook_type, color)
+    # FIXME: add post-commit somewhere
```

Comment:
since it takes 0 arguments, you can add it to the `pre-merge-commit` / `pre-commit` branch directly above your comment